ts/session/messages/outgoing/content/ContentMessage.ts

```diff
@@ -14,7 +14,6 @@ export abstract class ContentMessage extends Message {
    * this value can be used in all child classes
    */
   protected getDefaultTTL(): number {
-    // 1 day default for any other message
-    return 24 * 60 * 60 * 1000;
+    return window.CONSTANTS.TTL_DEFAULT_REGULAR_MESSAGE;
```

Comment:
can't we import this directly?
You'll also need to update `ttl()` in Pairing and device unpairing messages.